README.md

```diff
@@ -56,12 +60,62 @@ Statistics of the MIRACL corpora:
 | Telugu (te)     |       518,079 |        66,353 | [🤗](https://huggingface.co/datasets/miracl/miracl-corpus/tree/main/miracl-corpus-v1.0-te) | [🌏](https://archive.org/download/tewiki-20190201/tewiki-20190201-pages-articles-multistream.xml.bz2)
 | Thai (th)       |       542,166 |       128,179 | [🤗](https://huggingface.co/datasets/miracl/miracl-corpus/tree/main/miracl-corpus-v1.0-th) | [🌏](https://archive.org/download/thwiki-20190101/thwiki-20190101-pages-articles-multistream.xml.bz2)
 | Chinese (zh)    |     4,934,368 |     1,246,389 | [🤗](https://huggingface.co/datasets/miracl/miracl-corpus/tree/main/miracl-corpus-v1.0-zh) | [🌏](https://archive.org/download/zhwiki-20220301/zhwiki-20220301-pages-articles-multistream.xml.bz2)
+</details>
 
 
-## Relevance Judgments
+## Topics and Relevance Judgments
 
-Coming soon!
-See our release schedule [on the MIRACL website](http://miracl.ai).
+The topics and judgement of training and development set are now available on [HuggingFace Dataset](https://huggingface.co/datasets/miracl/miracl)! It is now hosted as a [gated dataset](https://huggingface.co/docs/hub/datasets-gated), where you need to share the name, affiliation and email to get access to the dataset. This is for the ease of announcing any future update about the dataset, and we will not disclose the information to any third-party.
```

Comment:
"It is hosted... third-party."

I don't think there's a need for such a detailed explanation?